packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -183,22 +183,21 @@ export class TreeItem extends FASTElement {
                 break;
             case keyCodeArrowDown:
                 // preventDefault to ensure we don't scroll the page
-                e.preventDefault();
                 this.focusNextNode(1);
                 break;
             case keyCodeArrowUp:
                 // preventDefault to ensure we don't scroll the page
-                e.preventDefault();
                 this.focusNextNode(-1);
                 break;
             case keyCodeEnter:
                 // In single-select trees where selection does not follow focus (see note below),
                 // the default action is typically to select the focused node.
                 this.handleSelected(e);
                 break;
+            default:
+                return true;
         }
-
-        return true;
+        return false;
```

Comment:
`return false` prevents default, stops propagation, and stops any further function execution.

Since this is the end of the function anyway, it should either just `return` (void) or not have a final return statement.

Reply:
Ah I thought of that, but ended up adding the statement anyway. I'll remove it! Thanks! :D

Reply:
Here's the piece of code you all want to look at: https://github.com/microsoft/fast/blob/master/packages/web-components/fast-element/src/templating/binding.ts#L368

Reply:
Well this is embarrassing 😳 the `return false` behavior I described is only relevant in the context of jQuery's `.on()` events, and may have been a thing for really old browsers (like, pre-2009). Returning false in a current-day event handler does nothing special. Either way, removing it here resolves exactly this sort of ambiguity, which is always good!